cmd/buildtools/openebs.go

```diff
@@ -78,3 +78,115 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		return nil
 	},
 }
+
+type addonComponent struct {
+	name                        string
+	wolfiPackageName            string
+	getWolfiPackageName         func(upstreamVersion string) string
+	upstreamVersionMakefileVar  string
+	upstreamVersionFlagOverride string
+}
+
+var openebsComponents = []addonComponent{
+	{
+		name:                        "openebs-provisioner-localpv",
+		wolfiPackageName:            "dynamic-localpv-provisioner",
+		upstreamVersionMakefileVar:  "OPENEBS_IMAGE_VERSION",
+		upstreamVersionFlagOverride: "openebs-version",
+	},
+	{
+		name:                        "openebs-linux-utils",
+		upstreamVersionMakefileVar:  "OPENEBS_IMAGE_VERSION",
+		upstreamVersionFlagOverride: "openebs-version",
+	},
+	{
+		name:                        "openebs-kubectl",
+		getWolfiPackageName:         func(upstreamVersion string) string { return "kubectl-" + upstreamVersion + "-default" },
+		upstreamVersionMakefileVar:  "OPENEBS_KUBECTL_IMAGE_VERSION",
+		upstreamVersionFlagOverride: "kubectl-version",
+	},
+}
+
+var updateOpenEBSImagesCommand = &cli.Command{
+	Name:      "openebs",
+	Usage:     "Updates the openebs images",
+	UsageText: environmentUsageText,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "openebs-version",
+			Usage: "The version of openebs to use to determine image versions",
+		},
+		&cli.StringFlag{
+			Name:  "kubectl-version",
+			Usage: "The version of kubectl to use to determine image versions",
+		},
+	},
+	Action: func(c *cli.Context) error {
+		logrus.Infof("updating openebs images")
+
+		if err := ApkoLogin(); err != nil {
+			return fmt.Errorf("failed to apko login: %w", err)
+		}
+
+		wolfiAPKIndex, err := GetWolfiAPKIndex()
+		if err != nil {
+			return fmt.Errorf("failed to get APK index: %w", err)
+		}
+
+		for _, component := range openebsComponents {
+			upstreamVersion, err := getAddonComponentUpstreamVersion(c, component)
```

Comment:
nit: can we make this a field or a method of the `addonComponent` struct and maybe call it `getUpstreamVersion`? seems inconsistent with how we're handling `getWolfiPackageName` for example.